src/main/include/CONSTANTS.h

```diff
@@ -5,3 +5,4 @@
 #pragma once
 
 constexpr double LEFT_TRIGGER_BARRIER = 0.5;
+constexpr double RIGHT_TRIGGER_BARRIER = 0.5;
```

Comment:
wait what's this for?

Reply:
Oh got it. Any reason why this is in a different file?

Reply:
Not sure... In general I'd like to have some organizational consistency -- we should decide how we want to organize things and put it into our style sheet.

My gut says:
ports -- rio ports, can ids, usb ids
robot constants -- mechanical constants like the number of modules, the physical location of camera, etc.
control constants -- sysid constants, PID constants, sensor values
human control constants -- deadband tuning, trigger barriers, etc.

I'm probably missing some there. Alternatively, subsystem specific constant files -- though that may come with it's own pitfalls